language-support/ts/daml-ledger/index.ts

```diff
@@ -1112,10 +1120,12 @@ class Ledger {
   async create<T extends object, K, I extends string>(
     template: Template<T, K, I>,
     payload: T,
+    meta?: CommandMeta,
```

Comment:
Why would you pass `meta` to a create command? Just for the `workflowId`?

Reply:
Just consistency. For now it doesn't really make sense. But I expect we'll include workflow ids, command ids, … in this at some point and then just passing the same to all commands seems easier. Happy to remove it for now if you prefer.